electron/renderer.js

```diff
@@ -168,21 +167,20 @@ class Renderer {
             });
         }
 
-        // TODO this needs to be reimplemented
-        // // Create new bindings for `console` functions
-        // // Use default console[name] and also send IPC
-        // // log so we can log to stdout
-        // function bindConsole() {
-        //     for (const name in console) {
-        //         if (typeof console[name] === 'function') {
-        //             globalLoggers[name] = console[name];
-        //             console[name] = function(...args) {
-        //                 globalLoggers[name].apply(console, args);
-        //                 ipcRenderer.send(name, args);
-        //             }
-        //         }
-        //     }
-        // }
+        // Create new bindings for `console` functions
+        // Use default console[name] and also send IPC
+        // log so we can log to stdout
+        function bindConsole() {
+            for (const name in console) {
+                if (typeof console[name] === 'function') {
+                    globalLoggers[name] = console[name];
+                    console[name] = function(...args) {
+                        globalLoggers[name].apply(console, args);
```

Comment:
you probably want to add my changes in here as well so we don't crash the console if someone logs an object with a circular dependency
something like:
```
                         let depthLimitStr = str;
                         if(typeof depthLimitStr === "object") {
                             depthLimitStr = util.inspect(depthLimitStr, {depth: this.logDepth });
                         }
                         remote.process.stdout.write(depthLimitStr);
```